chanreg.md

```diff
@@ -1,228 +1,196 @@
-
-# About you and your staff
-Your NickServ account:
-Your relationship / affiliation with the community:
-NickServ of group contacts and if their status is hidden/public:
-
-# Channels and cloaks (see below)
-Channels you'd like to claim:
-(typically #communityname and #communityname-*)
-Would you like ONLY group contacts to be able to register these channels with
-ChanServ?:
-(The cloaks will be formatted like this:
-about/YourCommunityNameHere/name or about/YourCommunityNameHere/role/name)
-```
-
-group contacts are the members in your community that will be the official
-interface between you and Libera.Chat.
-They are the ones who can claim channels in your namespace(s) and request
-cloaks for your members/users.
-You can tell us whether these should be visible to the public or not.
-If you need specific rules or abilities for your group contacts,
-check with our team to get the option that fits your structure best.
-
-Please send this information as an e-mail to <projects@libera.chat>, our team
-will then get in touch with you.
-
-### Benefits of a community
-
-Registered communities can profit from the following benefits:
-
-- Channels in their namespace can be claimed (with `/msg ChanServ CLAIM`)
-- Optional cloaks are available for members.
-  These replace the hostname part with `@about/yourcommunity/custompart`,
-  and can be requested on `#libera-communities`.
-
-## The Projects & Community Team
-
-The Projects & Community Team will act as your liaisons during the project
-registration or topic claiming process and throughout your tenure on the
-Libera.Chat network. If you wish to discuss the registration process, find out
-if Libera.Chat could be a good fit for your project or register your project,
-please feel free to drop any of the
-[team members](/about#projects-and-community-team) a line on IRC. If none are
-online, feel free to reach out to staff in general and they will forward your
-request to the team, which will get back to you as soon as possible. Once you
-have discussed your project with a staff member, you'll be asked to email
-<projects@libera.chat> with the details of your request as per the above.
+A formal community registration is done for on-topic community channels, and
+does not require the registrant to have an official claim to the channel's
+name. Libera.Chat staff will assist in determining which type of official
+registration is appropriate for your project or community.
+
+### Benefits of group registration
+
+On-topic projects and communities who have formally registered with
+Libera.Chat have extra tools at their disposal to help manage their
+IRC presence:
+
+- Your official liaisons with us, known as group contacts, will be invited to
+  `#libera-communities` where they can more easily reach Libera.Chat staff and
+  can be involved in discussions concerning potential future directions and
+  improvements for Libera.Chat.
+- Ownership over their group's names or name prefixes. For example,
```

Comment:
Can we use something like _Control_ or _Assignment_ rather than _Ownership_? We're not selling anything and things can be reassigned without compensation.